indexed_gzip/indexed_gzip.pyx

```diff
@@ -406,7 +406,8 @@ cdef class _IndexedGzipFile:
                               window_size=window_size,
                               readbuf_size=readbuf_size,
                               flags=flags):
-                raise ZranError('zran_init returned error')
+                raise ZranError('zran_init returned error (file: '
+                                '{})'.format(self.filename or 'n/a'))
```

Comment:
LGTM, although sometimes we might get `fileobj`s that have the `name` property defined. What about something like:

```suggestion
                name = self.filename or getattr(self.pyfid, 'name', 'n/a')
                raise ZranError('zran_init returned error (file: '
                                '{})'.format(name))
```